src/libs/ReportActionsUtils.ts

```diff
@@ -214,15 +214,53 @@ function getSortedReportActions(reportActions: ReportAction[] | null, shouldSort
         return (first.reportActionID < second.reportActionID ? -1 : 1) * invertedMultiplier;
     });
 
-    // If shouldMarkTheFirstItemAsNewest is true, label the first reportAction as isNewestReportAction
-    if (shouldMarkTheFirstItemAsNewest && sortedActions?.length > 0) {
-        sortedActions[0] = {
-            ...sortedActions[0],
-            isNewestReportAction: true,
-        };
+    return sortedActions;
+}
+
+/**
+ * Returns the range of report actions from the given array which include current id
+ * the range is consistent
+ *
+ * param {ReportAction[]} array
```

Comment:
No need for these JSDoc params since this file is using typescript now.

Reply:
👍🏻